src/app/shared/object-grid/admin-search-result-grid-element/item-search-result/item-admin-search-result-grid-element.component.html

```diff
@@ -0,0 +1,41 @@
+<ng-template dsListableObject>
+</ng-template>
+<div #badges class="position-absolute ml-1">
+    <div *ngIf="dso && !dso.isDiscoverable" class="private-badge">
+        <span class="badge badge-danger">{{ "admin.search.item.private" | translate }}</span>
+    </div>
+    <div *ngIf="dso && dso.isWithdrawn" class="withdrawn-badge">
+        <span class="badge badge-warning">{{ "admin.search.item.withdrawn" | translate }}</span>
+    </div>
+</div>
+<ul #buttons class="list-group list-group-flush">
+  <li class="list-group-item d-flex justify-content-between">
+    <a class="btn btn-light btn-sm my-1 edit-link" [routerLink]="[getEditPath()]" [title]="'admin.search.item.edit' | translate">
```

Comment:
I noticed the list of actions here is repeated in item-admin-search-result-list-element.component.html
I would rather expect the list of buttons to be part of a separate component and have item-admin-search-result-grid-element.component.html and item-admin-search-result-list-element.component.html only apply the style changes.
The same applies for the duplication in the ts